app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ReviewsController < ApplicationController
+  before_action :require_user_logged_in
+
+  def create
+    @review = Review.new(params.require(:review).permit(:rating, :review))
+    @video = Video.find(params[:video_id])
+    @review.video = @video
+    @review.user = current_user
+    if @review.save
+      flash[:success] = "You have reviewed #{@video.title}!"
+      redirect_to video_path(@video)
+    else
+      unless @review.errors.messages[:user_id]
```

Comment:
Generally, if you have both a happy and sad path, you should start with the happy path (`if`) instead of the sad path (`unless`). It makes the code more readable.